client/src/components/Playground/components/ColabList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {compose} from 'redux'
+import {connect} from 'react-redux'
+import {withDataProviders} from 'data-provider'
+import {ListGroup} from 'reactstrap'
+import type {Colab, State} from '../../../state'
+import {colabsSelector} from '../../../selectors'
+import {colabsProvider} from '../../../dataProviders/dataDataProviders'
+import ColabDetail from './ColabDetail'
+
+export type ColabListProps = {
+  colabs: Colab[],
+}
+
+const ColabList = ({colabs}: ColabListProps) => {
+  return (
+    <>
+      <h2 className="report-list-title">Zoznam reportov</h2>
+      <ListGroup>
+        {colabs.map((colab, index) => (
+          <ColabDetail colab={colab} key={index} />
```

Comment:
Generally, it's a bad practice to use array index as a key (because of unnecessary rerenders when insterting/removing elements). Are collab names unique? If not, the it's ok to use index I guess...

Reply:
There is no guarantee that the name of colab is unique... Although it makes sense.

Reply:
Ok then...